src/layerInfos/IsbTopicInfo/IsbTopicInfo.js

```diff
@@ -50,6 +51,13 @@ const IsbTopicInfo = ({ language }) => {
         <br />
         <a href="mailto:netzzugang@sbb.ch">netzzugang@sbb.ch</a>.
       </p>
+      <hr />
+      <p>
+        <DataLink
+          fullWidth={false}
+          href="https://data.sbb.ch/explore/dataset/infrastrukturbetreiberinnen/information/"
```

Comment:
I think we have cy test for data link.  can you add test for this one?

Reply:
Done in https://github.com/geops/trafimage-maps/pull/1074